assignments/objects.js

```diff
@@ -49,16 +111,38 @@ const example = {
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-const parent = {}
+const parent = {
+  "name":"Susan",
+  "age":70,
+  "speak":function(){
+    console.log(`I am ${this.name}`);
+  },
+  "child":{
+    "name":"George",
+    "age":50,
+    "speak":function(){
+      console.log(`I am ${this.name}`);
+    },
+    "grandChild":{
+      "name":"Sam",
+      "age":30,
+      "speak":function(){
+        console.log(`I am ${this.name}`);
+      }
+    }
+  }
+}
 
 // Log the parent object's name
-
+console.log(parent);
```

Comment:
It looks like you logged the entire `parent`  object instead of its `name` property here.